### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CrowRising**
# Description

Made a starting ReadMe, 
logo needs to be added via link, 
Installation needs to be added (I kept my original in case it was helpful to have for the markdown)
Screenshots and/or giphy to be added as well

Closes# (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [ ] Refactoring

# How Has This Been Tested?

Not tested

- [ ] Test A
- [ ] Test B

# UI/UX Changes: 

(Please include screenshot or gif)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Elise-Jones — APPROVED** [review]
Thank you so much for doing this Crow.

**Review by jmartin777 — APPROVED** [review]
THIS LOOKS GREAT CROW!!!!1 Thank You so much for putting this togethor!

File: README.md [hunk 1]
```diff
@@ -1,70 +1,139 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+[![Contributors](https://img.shields.io/github/contributors/In-The-Field/In_The_Field_FE.svg)](https://github.com/In-The-Field/In_The_Field_FE/graphs/contributors)
+[![Forks](https://img.shields.io/github/forks/In-The-Field/In_The_Field_FE.svg)](https://github.com/In-The-Field/In_The_Field_FE/forks)
+[![Stargazers](https://img.shields.io/github/stars/In-The-Field/In_The_Field_FE.svg)](https://githuB.com/In-The-Field/In_The_Field_FE/stargazers)
+[![Issues](https://img.shields.io/github/issues/In-The-Field/In_The_Field_FE.svg)](https://github.com/In-The-Field/In_The_Field_FE/issues)
+[![CircleCI Status](https://circleci.com/gh/facebook/react.svg?style=shield)](https://circleci.com/gh/facebook/react)
```
Comment (andreasorensen) [inline README.md:5]: Thank you for adding this!

**Review by andreasorensen — APPROVED** [review]
This is amazing. Thank you for setting this up Crow! I'm definitely going to use this template going forward!

**Pull request merged**